frontend/src/views/instructors/editable-instructors-list.js

```diff
@@ -42,7 +58,38 @@ function EditableInstructorsList(props) {
         );
     }
 
+    const instructorCurrentlyAssignedHash = {};
+    (positions || []).map((position) =>
+        position.instructors.map(
+            (instructor) =>
+                (instructorCurrentlyAssignedHash[instructor.id] = true)
+        )
+    );
+
     const columns = [
+        {
+            Header: (
+                <FaTrash className="delete-instructor-column-header-icon" />
+            ),
+            Cell: (props) => {
+                // Renders a custom cell with a delete button if the instructor is not currently assigned to any position, otherwise nothing
+                const instructor = props.original;
+                const disabled =
+                    instructorCurrentlyAssignedHash[instructor.id] === true; // if the instructor is currently assigned, don't return a delete button
```

Comment:
Prefer comments on the line before over comments at the end of a line.

Reply:
We should add this to prettify rules.